common/src/main/java/com/abrenoch/hyperiongrabber/common/HyperionScreenService.java

```diff
@@ -40,7 +40,7 @@
     public static final String EXTRA_RESULT_CODE = BASE + "EXTRA_RESULT_CODE";
     private static final int NOTIFICATION_ID = 1;
     private static final int NOTIFICATION_EXIT_INTENT_ID = 2;
-
+    private static final int SUCCESSFUL_CONNECTION_NOTIFY_DELAY_MS = 100;
```

Comment:
The activity communicating to the service that it's ready would be much more elegant than a delay